README.md

```diff
@@ -66,9 +66,50 @@ contract MyContract is Ownable {
+- **lifecycle** - A collection of base contracts used to manage the existence and behavior of your contracts and their funds.
+- **math** - Libraries with safety checks used to prevent against overflows and underflows.
+- **mocks** - A collection of abstract contracts that can be used as the foundation for your own custom implementations. The mock contracts demonstrate how OpenZeppelin’s secure base contracts can be used with multiple inheritance.
+- **ownership** - A collection of smart contracts that can be used to manage contract and token ownership
+	- **rbac** - A library used to manage addresses assigned to different user roles and an example RBAC interface that demonstrates how to handle setters and getters for roles and addresses.
+
+- **payment** - A collection of smart contracts that can be used to manage payments through escrow arrangements, withdrawals, and claims. Includes support for both single payees and multiple payees.
+- **proposals** - A collection of smart contracts that reflect community Ethereum Improvement Proposals (EIPs). The smart contracts here are under consideration for inclusion in OpenZeppelin. They are not recommended for use until the community has thoroughly evaluated and approved them.
+- **token** - A collection of approved ERC standard tokens -- their interfaces and implementations.
+- **ERC20** - A standard interface for fungible tokens:
+	- *Interfaces* - Includes the ERC-20 token standard basic interface. I.e., what the contract’s ABI can represent.
+	- *Implementations* - Includes ERC-20 token implementations that include all required and some optional ERC-20 functionality.
+- **ERC721** - A standard interface for non-fungible tokens 
+	- *Interfaces* - Includes the ERC-721 token standard basic interface. I.e., what the contract’s ABI can represent.
+	- *Implementations* - Includes ERC-721 token implementations that include all required and some optional ERC-721 functionality.
+
+## Tests
+Unit test are critical to the OpenZeppelin framework. They help ensure code quality and mitigate against security vulnerabilities. The directory structure within the `/tests` directory corresponds to the `/contracts` directory. OpenZeppelin uses Mocha’s JavaScript testing framework and Chai’s assertion library. To learn more about how to tests are structured, please reference OpenZeppelin’s Testing Guide.
+
+## How To Use And Modify OpenZeppelin Contracts
+When using OpenZeppelin to build your own distributed applications, for security reasons we encourage you NOT to modify the framework’s base contracts, libraries, and interfaces. In order to leverage and extend their functionality, we encourage you to inherit from them.
+
+The Solidity programming language supports multiple inheritance. This is very powerful yet it can also be confusing. The more complexity you introduce to your distributed applications through multiple inheritance, the greater your application’s attack surface is.
```

Comment:
nit: can we add a `:` in between these two sentences, like: `This is very powerful, but can also be confusing: the more complexity you introduce to your distributed applications...`